test/e2e/network/dns_configmap.go

```diff
@@ -201,12 +201,16 @@ func (t *dnsNameserverTest) run() {
 	if t.name == "coredns" {
 		t.setConfigMap(&v1.ConfigMap{Data: map[string]string{
 			"Corefile": fmt.Sprintf(`.:53 {
+        errors
+        health
         kubernetes cluster.local in-addr.arpa ip6.arpa {
            pods insecure
            upstream
            fallthrough in-addr.arpa ip6.arpa
         }
+        prometheus :9153
         proxy . /etc/resolv.conf
+        cache 30
```

Comment:
Humm...Instead of hardcoding the config value here, can we fetch the configmap directly before test? In that way we are off the burden of keeping these consistent with whatever is in the CoreDNS deployment configuration.

Similar to:
https://github.com/kubernetes/kubernetes/blob/7076eed0b9c965444ffe4deaecad2aaac66876fb/test/e2e/autoscaling/dns_autoscaling.go#L65-L67

Reply:
Done.